contracts/multisig-prover/src/execute.rs

```diff
@@ -129,31 +132,50 @@ fn get_verifiers_info(deps: &DepsMut, config: &Config) -> Result<VerifiersInfo,
             msg: to_json_binary(&active_verifiers_query)?,
         }))?;
 
-    let participants = verifiers
-        .clone()
+    let min_num_verifiers = deps
```

Comment:
Not necessarily for this PR, but I think this should be configurable per connection. We could store this in the config at instantiation, and provide some way to update it. It could default to what's stored in the service registry if unspecified.